src/ui-kit/components/IconButton/IconButton.styled.js

```diff
@@ -0,0 +1,35 @@
+import IconButton from '@mui/material/IconButton';
+import styled from '@emotion/styled';
+
+export const StyledIconButton = styled(IconButton)`
+  padding-left: ${(p) => (p.pLeft ? p.pLeft : '3.13px')};
+  padding-right: ${(p) => (p.pRight ? p.pRight : '4.71px')};
+  padding-top: ${(p) => (p.pTop ? p.pTop : '3.13px')};
+  padding-bottom: ${(p) => (p.pBottom ? p.pBottom : '4.71px')};
+  stroke: ${(p) => (p.defaultStroke ? p.defaultStroke : 'none')};
```

Comment:
Does it work?

Reply:
We have different paddings and to set them for each individual case. Stroke set another color of black if we need